test/e2e/backup-restore/backup_restore_test.go

```diff
@@ -32,16 +32,16 @@ func TestMain(m *testing.M) {
 }
 
 func TestBackupRestore(t *testing.T) {
-	t.Run(string(v1.ServiceTypeDataGrid), testBackupRestore(datagridService))
-	t.Run(string(v1.ServiceTypeCache), testBackupRestore(cacheService))
+	t.Run(string(v1.ServiceTypeDataGrid), testBackupRestore(datagridService, 2))
+	t.Run(string(v1.ServiceTypeDataGrid)+"NoAuth", testBackupRestore(datagridServiceNoAuth, 1))
```

Comment:
Why we need extra Backup/Restore test without auth?

Reply:
It's to ensure that the zero capacity pod is configured correctly with/without identities and that the backup/restore controllers enable/disable auth as configured.

The `NoAuth` test only creates clusters with 1 pod, so it isn't as heavyweight as the normal backup tests. Similarly, I have only added the test for the DataGrid Service as the behaviour will be the same for the two service types.

Reply:
Thanks, make sense.